src/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card {
+  padding: 16px;
+  padding-top: 32px;
```

Comment:
The padding property is defined twice in the .card class. The first definition sets padding to 16px, and the second sets padding-top to 32px. This redundancy can lead to confusion or unexpected results. Consider removing one of these definitions or consolidating them if different values are intended for different sides.

Reply:
when I remove one of these paddings, BackstopJs curses at me and writes an error